app/assets/stylesheets/modules/mixins.scss

```diff
@@ -27,3 +27,14 @@
   padding-left: 3em;
   padding-right: 3em;
 }
+
+@mixin link-border {
+  border-bottom: 2px solid transparent;
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.